app/controllers/user/applications_controller.rb

```diff
@@ -0,0 +1,16 @@
+class User::ApplicationsController < ApplicationController
+	before_action :ensure_either_broker_or_ff
+
+	def index
+		if current_user.bids.exists?
+			@applied_bids = current_user.bids.all
```

Comment:
Let's `includes(:job)` to prevent N + 1 :) I noticed that we query for `bid.job` in the views, so it's necessary :)